ui/src/components/SearchControls/components/SearchAggregations/SearchAggregationsCards.js

```diff
@@ -37,7 +28,7 @@ export default class SearchAggregationsCards extends Component {
           </List.Content>
           <List.Content>
             <Checkbox
-              label={label}
+              label={`${bucket.key}`}
```

Comment:
```suggestion
              label={bucket.key}
```
The bucket key is already a string, right?